flash_cards/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+# from django.utils.text import slugify
 
 # Create your models here.
+class FlashCard(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField(max_length=1000, null=True, blank=True)
+    deck = models.ForeignKey('Deck', on_delete=models.CASCADE)
```

Comment:
```suggestion
    deck = models.ForeignKey('Deck', related_name="flashcards", on_delete=models.CASCADE)
```
This should allow us to ask django which flashcards are associated with our deck, _as well as_ what you already had, which would allow us to ask django which deck is associated with our flashcards